db/seeds/9999_dev_only.rb

```diff
@@ -0,0 +1,27 @@
+# This file contains content previously in the working:setup task
+# which is useful in development environments for easy debugging.
+# It has been moved into seeds to streamline the process.
+
+if Rails.env.development?
+  require './lib/working_setup/standard_seeder'
+  # Barcode printers (These match some of the more frequently used printers,
+  # or at least the most frequently used when I first put this together)
+  plate = BarcodePrinterType.find_by!(name: '96 Well Plate')
+  tube = BarcodePrinterType.find_by!(name: '1D Tube')
+  BarcodePrinter.find_or_create_by!(name: 'g312bc2', barcode_printer_type: plate)
+  BarcodePrinter.find_or_create_by!(name: 'g311bc2', barcode_printer_type: plate)
+  BarcodePrinter.find_or_create_by!(name: 'g316bc',  barcode_printer_type: plate)
+  BarcodePrinter.find_or_create_by!(name: 'g317bc',  barcode_printer_type: plate)
+  BarcodePrinter.find_or_create_by!(name: 'g314bc',  barcode_printer_type: plate)
+  BarcodePrinter.find_or_create_by!(name: 'f225bc',  barcode_printer_type: plate)
+  BarcodePrinter.find_or_create_by!(name: 'g311bc1', barcode_printer_type: tube)
+
+  # Previous content of working:basic provides a few example studies
+  # and the admin user
+  seeder = WorkingSetup::StandardSeeder.new
+  seeder.user
+  seeder.study
+  seeder.study_b
+  seeder.project
+  seeder.supplier
+end
```

Comment:
The working setup version was adding 'seed' and 'tag_plates', do those need to be added to the list here?

Reply:
I decided to push tag plates out to the UAT actions, especially as they can be a little slow to create. I'll double check what seed was doing.

Reply:
Looks like seed did a few more things than I assumed. I'll extract out the still relevant bits. It was mostly generating plates for older pipelines, which I'll leave out, but also made a few robots.

Reply:
Ahh, so just don't pass in purposes and that'll do what I want I think. I'll add seed back in for now.

Reply:
I'll follow this up with moving a lot of this to record loaders

Reply:
Actually, decided to leave the single stock plate in here.

Reply:
What (if any) additional UAT actions do we need? Some of the ones we have need some more options (e.g. adding controls), but you can do most things now that you'd need for testing.

Reply:
Steve asked for one for a tagged plate. I've added a story to the backlog for that. Figured its worth keeping separate from the normal plate just to avoid overwhelming it with options.